src/PrizmMainProject/Languages/LanguageManager.cs

```diff
@@ -323,8 +323,9 @@ public IEnumerable<StringResource> EnumerateStringResources(System.Type stringRe
             return ret;
         }
 
-        public bool ApplyUsersLanguage(User user)
+        public bool ApplyUsersLanguage(User user, out int userLangIndex)
         {
+            userLangIndex = (user.UILanguage == null) ? indexDefault : cultures.FindIndex((lp) => { return lp.Culture.Name == user.UILanguage; });
             return (user.UILanguage == null) ? false : (user.UILanguage != CurrentCulture.Name && LoadTranslation(new CultureInfo(user.UILanguage)));
```

Comment:
If no translations are found, LoadTranslation method returns false, ApplyUsersLanguage returns false and 
that code
 this.CascadeChangeLanguage();
 languageBarListItem.DataIndex = userLangIndex; 
in PrizmApplicationXtraForm is not executed